oob/columns.lisp

```diff
@@ -4,7 +4,7 @@
   (STAMP :i32)
   (CT :i3)
   (CT_MAX :i3)
-  (DATA :array [8])
+  (DATA :array [9])
```

Comment:
@lorenzogentile404 We should add any missing type annotations, e.g. for the `(OOB_)DATA`.